diff --git a/lib/tjbot.js b/lib/tjbot.js
index 6a530c8..821ccfa 100644
--- a/lib/tjbot.js
+++ b/lib/tjbot.js
@@ -917,7 +917,7 @@ TJBot.prototype.recognizeObjectsInPhoto = function(filePath, classifier_ids) {
         var params = {
             images_file: fs.createReadStream(filePath),
             threshold: self.configuration.see.confidenceThreshold.object,
-            'Accept-Language': self.configuration.see.language
+            accept_language: self.configuration.see.language
         };
 
         if (classifier_ids != undefined) {
@@ -970,7 +970,7 @@ TJBot.prototype.recognizeTextInPhoto = function(filePath) {
         var params = {
             images_file: fs.createReadStream(filePath),
             threshold: self.configuration.see.confidenceThreshold.text,
-            'Accept-Language': self.configuration.see.language
+            accept_language: self.configuration.see.language
         };
 
         self._visualRecognition.recognizeText(params, function(err, response) {
